[2023-12-20 16:31:27,692: INFO: main: Welcome to our custom log]
[2023-12-27 12:11:21,118: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-27 12:11:21,121: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-27 12:11:21,123: INFO: common: created directory at: artifacts]
[2023-12-27 12:23:54,542: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-27 12:23:54,544: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-27 12:23:54,545: INFO: common: created directory at: artifacts]
[2023-12-27 12:23:54,547: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-27 12:23:54,548: INFO: 3525658959: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-12-27 12:25:48,691: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-27 12:25:48,693: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-27 12:25:48,695: INFO: common: created directory at: artifacts]
[2023-12-27 12:25:48,696: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-27 12:25:48,697: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-12-27 12:28:38,077: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:46:12,615: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-28 14:46:12,617: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-28 14:46:12,618: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 14:46:12,618: INFO: common: created directory at: artifacts]
[2024-02-28 14:46:12,618: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-28 14:46:12,619: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:46:14,745: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:46:14,746: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 8, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.extract_zip_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-02-28 14:52:58,888: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-28 14:52:58,889: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-28 14:52:58,890: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 14:52:58,890: INFO: common: created directory at: artifacts]
[2024-02-28 14:52:58,890: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-28 14:52:58,890: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:53:01,014: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:53:01,014: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 8, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.extract_zip_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-02-28 14:57:12,452: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-28 14:57:12,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-28 14:57:12,454: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 14:57:12,455: INFO: common: created directory at: artifacts]
[2024-02-28 14:57:12,455: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-28 14:57:12,455: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:57:15,005: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:57:15,401: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 12:51:25,643: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 12:51:25,650: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 12:51:25,654: INFO: common: created directory at: artifacts]
[2024-03-20 13:12:09,350: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:12:09,353: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:12:09,355: INFO: common: created directory at: artifacts]
[2024-03-20 13:14:33,159: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:14:33,162: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:14:33,164: INFO: common: created directory at: artifacts]
[2024-03-20 13:14:39,245: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:14:39,249: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:14:39,251: INFO: common: created directory at: artifacts]
[2024-03-20 13:18:05,515: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:18:05,518: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:18:05,520: INFO: common: created directory at: artifacts]
[2024-03-20 13:18:05,521: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 13:19:51,651: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:21:19,094: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:21:19,097: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:21:19,098: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:21:19,098: INFO: common: created directory at: artifacts]
[2024-03-20 21:21:19,099: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:21:19,099: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:21:20,792: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:21:21,237: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 21:21:21,237: INFO: main: *******************]
[2024-03-20 21:21:21,237: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-20 21:21:21,239: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:21:21,241: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:21:21,241: INFO: common: created directory at: artifacts]
[2024-03-20 21:21:21,241: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 21:21:21,620: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:21:21,818: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-20 21:48:37,671: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:48:37,674: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:48:37,676: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:48:37,676: INFO: common: created directory at: artifacts]
[2024-03-20 21:48:37,676: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:48:37,676: INFO: data_ingestion: Downloading data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:48:38,426: INFO: data_ingestion: Downloaded data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:48:38,788: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 21:48:38,789: INFO: main: *******************]
[2024-03-20 21:48:38,789: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-20 21:48:38,791: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:48:38,792: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:48:38,793: INFO: common: created directory at: artifacts]
[2024-03-20 21:48:38,793: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 21:48:39,191: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:48:39,395: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-20 21:51:17,419: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:51:17,422: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:51:17,424: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:51:17,424: INFO: common: created directory at: artifacts]
[2024-03-20 21:51:17,424: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:51:17,425: INFO: data_ingestion: Downloading data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:51:18,053: INFO: data_ingestion: Downloaded data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:51:18,053: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.extract_zip_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-03-20 21:52:38,451: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:52:38,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:52:38,456: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:52:38,456: INFO: common: created directory at: artifacts]
[2024-03-20 21:52:38,456: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:52:38,456: INFO: data_ingestion: Downloading data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:52:39,092: ERROR: main: Failed to retrieve file url:

	Cannot retrieve the public link of the file. You may need to change
	the permission to 'Anyone with the link', or have had many accesses.
	Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

You may still be able to access the file from the browser:

	https://drive.google.com/uc?/export=download&id=drive.usercontent.google.com

but Gdown can't. Please check connections and permissions.]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\gdown\download.py", line 253, in download
    url = get_url_from_gdrive_confirmation(res.text)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\gdown\download.py", line 54, in get_url_from_gdrive_confirmation
    raise FileURLRetrievalError(
gdown.exceptions.FileURLRetrievalError: Cannot retrieve the public link of the file. You may need to change the permission to 'Anyone with the link', or have had many accesses. Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 9, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 14, in main
    data_ingestion.download_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 31, in download_file
    raise e
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 26, in download_file
    gdown.download(prefix+file_id,zip_download_dir)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\gdown\download.py", line 264, in download
    raise FileURLRetrievalError(message)
gdown.exceptions.FileURLRetrievalError: Failed to retrieve file url:

	Cannot retrieve the public link of the file. You may need to change
	the permission to 'Anyone with the link', or have had many accesses.
	Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

You may still be able to access the file from the browser:

	https://drive.google.com/uc?/export=download&id=drive.usercontent.google.com

but Gdown can't. Please check connections and permissions.
[2024-03-20 21:53:33,149: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:53:33,151: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:53:33,153: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:53:33,153: INFO: common: created directory at: artifacts]
[2024-03-20 21:53:33,153: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:53:33,153: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:53:40,686: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:53:41,063: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 21:53:41,063: INFO: main: *******************]
[2024-03-20 21:53:41,064: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-20 21:53:41,066: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:53:41,068: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:53:41,068: INFO: common: created directory at: artifacts]
[2024-03-20 21:53:41,068: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 21:53:41,450: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:53:41,677: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-20 22:07:00,636: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:07:00,644: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:07:00,646: INFO: common: created directory at: artifacts]
[2024-03-20 22:09:24,844: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:09:24,847: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:09:24,849: INFO: common: created directory at: artifacts]
[2024-03-20 22:10:37,624: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:10:37,627: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:10:37,629: INFO: common: created directory at: artifacts]
[2024-03-20 22:13:05,373: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:13:05,377: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:13:05,379: INFO: common: created directory at: artifacts]
[2024-03-20 22:19:34,768: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:19:34,771: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:19:34,773: INFO: common: created directory at: artifacts]
[2024-03-20 22:19:34,775: INFO: common: created directory at: artifacts\training]
[2024-03-20 23:11:46,271: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 23:11:46,275: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 23:11:46,276: INFO: common: created directory at: artifacts]
[2024-03-20 23:11:46,278: INFO: common: created directory at: artifacts\training]
[2024-03-21 00:31:54,057: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-21 00:31:54,060: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 00:31:54,062: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 00:31:54,062: INFO: common: created directory at: artifacts]
[2024-03-21 00:31:54,063: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-21 00:31:54,064: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-21 00:31:59,632: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-21 00:32:00,048: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-21 00:32:00,048: INFO: main: *******************]
[2024-03-21 00:32:00,048: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-21 00:32:00,051: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 00:32:00,053: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 00:32:00,053: INFO: common: created directory at: artifacts]
[2024-03-21 00:32:00,053: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-21 00:32:00,439: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-21 00:32:00,642: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-21 00:32:00,642: INFO: main: *******************]
[2024-03-21 00:32:00,643: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-21 00:32:00,646: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 00:32:00,647: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 00:32:00,648: INFO: common: created directory at: artifacts]
[2024-03-21 00:32:00,648: INFO: common: created directory at: artifacts\training]
[2024-03-21 00:38:45,956: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-21 10:13:23,413: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 10:13:23,421: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 10:13:23,424: INFO: common: created directory at: artifacts]
[2024-03-21 10:13:43,998: INFO: common: json file saved at: scores.json]
[2024-03-21 10:13:51,899: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-21 10:13:53,026: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp7j0ogdhj\model\data\model\assets]
[2024-03-21 11:06:58,561: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 11:06:58,565: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:06:58,566: INFO: common: created directory at: artifacts]
[2024-03-21 11:06:58,568: INFO: common: created directory at: artifacts\training]
[2024-03-21 12:53:39,946: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 12:53:39,950: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 12:53:39,951: INFO: common: created directory at: artifacts]
[2024-03-21 12:53:39,954: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-21 12:53:39,955: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-31 15:16:40,149: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 15:16:40,152: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 15:16:40,154: INFO: common: created directory at: artifacts]
[2024-03-31 15:16:40,155: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-31 15:16:40,157: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-31 15:19:30,915: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-31 15:39:30,549: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 15:39:30,552: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 15:39:30,554: INFO: common: created directory at: artifacts]
[2024-03-31 15:39:30,556: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-31 15:39:30,981: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-31 16:08:29,867: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 16:08:29,870: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 16:08:29,872: INFO: common: created directory at: artifacts]
[2024-03-31 16:08:29,873: INFO: common: created directory at: artifacts\training]
[2024-03-31 17:54:21,287: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 17:54:21,292: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 17:54:21,294: INFO: common: created directory at: artifacts]
[2024-03-31 17:54:21,296: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-31 17:54:21,875: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-31 18:01:43,508: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 18:01:43,511: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 18:01:43,513: INFO: common: created directory at: artifacts]
[2024-03-31 18:01:43,514: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-31 18:01:43,891: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-31 18:02:01,783: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 18:02:01,786: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 18:02:01,788: INFO: common: created directory at: artifacts]
[2024-03-31 18:02:01,789: INFO: common: created directory at: artifacts\training]
[2024-03-31 22:13:13,479: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 22:13:13,482: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 22:13:13,483: INFO: common: created directory at: artifacts]
[2024-03-31 22:14:40,218: INFO: common: json file saved at: scores.json]
[2024-03-31 22:14:44,087: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-31 22:14:44,676: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp235_xcur\model\data\model\assets]
[2024-03-31 22:25:19,798: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 22:25:19,803: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 22:25:19,805: INFO: common: created directory at: artifacts]
[2024-03-31 22:26:45,979: INFO: common: json file saved at: scores.json]
[2024-03-31 22:26:50,423: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-31 22:26:51,022: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpsq7nbjb3\model\data\model\assets]
[2024-04-01 21:35:04,684: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:35:04,687: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:35:04,689: INFO: common: created directory at: artifacts]
[2024-04-01 21:35:04,691: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-01 21:35:04,692: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-01 21:35:38,197: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-01 21:37:39,147: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:37:39,150: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:37:39,152: INFO: common: created directory at: artifacts]
[2024-04-01 21:37:39,153: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-01 21:37:39,549: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-01 21:38:25,968: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:38:25,971: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:38:25,973: INFO: common: created directory at: artifacts]
[2024-04-01 21:38:25,975: INFO: common: created directory at: artifacts\training]
[2024-04-01 21:39:27,931: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:39:27,934: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:39:27,935: INFO: common: created directory at: artifacts]
[2024-04-01 21:39:27,937: INFO: common: created directory at: artifacts\training]
[2024-04-01 21:40:37,542: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:40:37,546: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:40:37,547: INFO: common: created directory at: artifacts]
[2024-04-01 21:40:37,549: INFO: common: created directory at: artifacts\training]
[2024-04-01 21:42:31,082: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:42:31,085: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:42:31,087: INFO: common: created directory at: artifacts]
[2024-04-01 21:42:31,088: INFO: common: created directory at: artifacts\training]
[2024-04-01 22:03:22,692: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:03:22,696: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:03:22,697: INFO: common: created directory at: artifacts]
[2024-04-01 22:03:22,699: INFO: common: created directory at: artifacts\training]
[2024-04-01 22:09:30,909: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:09:30,912: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:09:30,914: INFO: common: created directory at: artifacts]
[2024-04-01 22:09:30,915: INFO: common: created directory at: artifacts\training]
[2024-04-01 22:49:49,199: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:49:49,202: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:49:49,204: INFO: common: created directory at: artifacts]
[2024-04-01 22:49:49,205: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-01 22:49:49,207: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-01 22:49:55,197: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-01 22:50:05,772: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:50:05,776: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:50:05,778: INFO: common: created directory at: artifacts]
[2024-04-01 22:50:05,779: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-01 22:50:06,213: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-01 22:52:10,950: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:52:10,954: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:52:10,956: INFO: common: created directory at: artifacts]
[2024-04-01 22:52:10,957: INFO: common: created directory at: artifacts\training]
[2024-04-02 00:21:00,501: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:21:00,504: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:21:00,506: INFO: common: created directory at: artifacts]
[2024-04-02 00:22:26,144: INFO: common: json file saved at: scores.json]
[2024-04-02 00:22:29,593: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 00:22:30,220: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp9rl9t5o5\model\data\model\assets]
[2024-04-02 00:45:12,053: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 00:45:12,055: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:45:12,057: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:45:12,057: INFO: common: created directory at: artifacts]
[2024-04-02 00:45:12,057: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 00:45:12,058: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 00:45:18,496: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 00:45:18,919: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 00:45:18,919: INFO: main: *******************]
[2024-04-02 00:45:18,919: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 00:45:18,921: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:45:18,923: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:45:18,923: INFO: common: created directory at: artifacts]
[2024-04-02 00:45:18,923: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 00:45:19,321: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 00:45:19,528: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 00:45:19,528: INFO: main: *******************]
[2024-04-02 00:45:19,528: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 00:45:19,530: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:45:19,532: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:45:19,532: INFO: common: created directory at: artifacts]
[2024-04-02 00:45:19,533: INFO: common: created directory at: artifacts\training]
[2024-04-02 00:46:15,389: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 00:46:15,389: INFO: main: *******************]
[2024-04-02 00:46:15,389: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 00:46:15,391: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:46:15,393: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:46:15,393: INFO: common: created directory at: artifacts]
[2024-04-02 00:46:24,674: INFO: common: json file saved at: scores.json]
[2024-04-02 00:46:24,675: INFO: common: json file saved at: scores.json]
[2024-04-02 00:46:24,675: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 01:43:29,942: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 01:43:29,944: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:43:29,946: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:43:29,947: INFO: common: created directory at: artifacts]
[2024-04-02 01:43:29,947: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 01:43:29,947: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 01:43:35,524: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 01:43:36,100: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 01:43:36,100: INFO: main: *******************]
[2024-04-02 01:43:36,100: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 01:43:36,102: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:43:36,104: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:43:36,105: INFO: common: created directory at: artifacts]
[2024-04-02 01:43:36,105: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 01:43:36,509: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 01:43:36,713: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 01:43:36,713: INFO: main: *******************]
[2024-04-02 01:43:36,714: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 01:43:36,716: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:43:36,717: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:43:36,718: INFO: common: created directory at: artifacts]
[2024-04-02 01:43:36,718: INFO: common: created directory at: artifacts\training]
[2024-04-02 01:44:35,003: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 01:44:35,003: INFO: main: *******************]
[2024-04-02 01:44:35,003: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 01:44:35,006: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:44:35,007: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:44:35,008: INFO: common: created directory at: artifacts]
[2024-04-02 01:44:44,708: INFO: common: json file saved at: scores.json]
[2024-04-02 01:44:44,708: INFO: common: json file saved at: scores.json]
[2024-04-02 01:44:48,224: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 01:44:48,796: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp1f2wbfg9\model\data\model\assets]
[2024-04-02 01:45:34,431: ERROR: main: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    model_evalution.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 20, in main
    evaluation.log_into_mlflow()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 72, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model", registered_model_name="VGG16Model")
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 562, in log
    mlflow.register_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 77, in register_model
    raise e
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 68, in register_model
    create_model_response = client.create_registered_model(name)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1743, in create_registered_model
    return self._get_registry_client().create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\client.py", line 63, in create_registered_model
    return self.store.create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\rest_store.py", line 75, in create_registered_model
    response_proto = self._call_endpoint(CreateRegisteredModel, req_body)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\base_rest_store.py", line 42, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: ''
[2024-04-02 14:20:50,545: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 14:20:50,548: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:20:50,549: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:20:50,550: INFO: common: created directory at: artifacts]
[2024-04-02 14:20:50,550: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 14:20:50,550: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 14:20:55,820: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 14:20:56,196: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 14:20:56,196: INFO: main: *******************]
[2024-04-02 14:20:56,196: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 14:20:56,198: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:20:56,200: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:20:56,200: INFO: common: created directory at: artifacts]
[2024-04-02 14:20:56,201: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 14:20:56,580: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 14:20:56,780: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 14:20:56,780: INFO: main: *******************]
[2024-04-02 14:20:56,781: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 14:20:56,783: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:20:56,784: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:20:56,785: INFO: common: created directory at: artifacts]
[2024-04-02 14:20:56,785: INFO: common: created directory at: artifacts\training]
[2024-04-02 14:21:52,806: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 14:21:52,806: INFO: main: *******************]
[2024-04-02 14:21:52,806: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 14:21:52,808: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:21:52,810: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:21:52,810: INFO: common: created directory at: artifacts]
[2024-04-02 14:22:02,161: INFO: common: json file saved at: scores.json]
[2024-04-02 14:22:02,161: INFO: common: json file saved at: scores.json]
[2024-04-02 14:22:05,561: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 14:22:06,162: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpjv7rpfsv\model\data\model\assets]
[2024-04-02 14:22:45,917: ERROR: main: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    model_evalution.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 20, in main
    evaluation.log_into_mlflow()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 72, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model", registered_model_name="VGG16Model")
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 562, in log
    mlflow.register_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 77, in register_model
    raise e
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 68, in register_model
    create_model_response = client.create_registered_model(name)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1743, in create_registered_model
    return self._get_registry_client().create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\client.py", line 63, in create_registered_model
    return self.store.create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\rest_store.py", line 75, in create_registered_model
    response_proto = self._call_endpoint(CreateRegisteredModel, req_body)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\base_rest_store.py", line 42, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: ''
[2024-04-02 15:10:27,228: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:10:27,231: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:10:27,233: INFO: common: created directory at: artifacts]
[2024-04-02 15:10:37,228: INFO: common: json file saved at: scores.json]
[2024-04-02 15:10:41,149: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 15:10:41,747: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpg7fzy0tj\model\data\model\assets]
[2024-04-02 15:44:00,735: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 15:44:00,738: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:44:00,739: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:44:00,740: INFO: common: created directory at: artifacts]
[2024-04-02 15:44:00,740: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 15:44:00,741: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 15:44:06,039: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 15:44:06,457: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 15:44:06,457: INFO: main: *******************]
[2024-04-02 15:44:06,457: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 15:44:06,460: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:44:06,461: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:44:06,461: INFO: common: created directory at: artifacts]
[2024-04-02 15:44:06,462: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 15:44:06,854: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 15:44:07,049: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 15:44:07,050: INFO: main: *******************]
[2024-04-02 15:44:07,050: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 15:44:07,052: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:44:07,054: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:44:07,054: INFO: common: created directory at: artifacts]
[2024-04-02 15:44:07,054: INFO: common: created directory at: artifacts\training]
[2024-04-02 15:45:02,957: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 15:45:02,957: INFO: main: *******************]
[2024-04-02 15:45:02,957: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 15:45:02,960: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:45:02,961: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:45:02,961: INFO: common: created directory at: artifacts]
[2024-04-02 15:45:12,448: INFO: common: json file saved at: scores.json]
[2024-04-02 15:45:12,448: INFO: common: json file saved at: scores.json]
[2024-04-02 15:45:15,995: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 15:45:16,564: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpilo3chyr\model\data\model\assets]
[2024-04-02 15:45:57,174: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 16:36:59,859: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-04-02 16:36:59,862: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:36:59,864: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:36:59,864: INFO: common: created directory at: artifacts]
[2024-04-02 16:36:59,865: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 16:36:59,865: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 16:37:06,616: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 16:37:07,036: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x=========x]
[2024-04-02 16:37:13,546: INFO: stage_02_prepare_base_model: *******************]
[2024-04-02 16:37:13,546: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 16:37:13,548: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:37:13,551: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:37:13,551: INFO: common: created directory at: artifacts]
[2024-04-02 16:37:13,551: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 16:37:13,949: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 16:37:14,182: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 16:37:18,962: INFO: stage_03_model_training: *******************]
[2024-04-02 16:37:18,962: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-02 16:37:18,965: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:37:18,967: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:37:18,967: INFO: common: created directory at: artifacts]
[2024-04-02 16:37:18,968: INFO: common: created directory at: artifacts\training]
[2024-04-02 16:38:17,937: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 16:38:23,626: INFO: stage_04_model_evaluation: *******************]
[2024-04-02 16:38:23,626: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 16:38:23,629: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:38:23,630: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:38:23,631: INFO: common: created directory at: artifacts]
[2024-04-02 16:38:33,663: INFO: common: json file saved at: scores.json]
[2024-04-02 16:38:33,663: INFO: common: json file saved at: scores.json]
[2024-04-02 16:38:33,664: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 17:41:49,010: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 17:41:49,012: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 17:41:49,014: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 17:41:49,014: INFO: common: created directory at: artifacts]
[2024-04-02 17:41:49,014: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 17:41:49,015: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 17:41:54,751: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 17:41:55,130: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 17:41:55,131: INFO: main: *******************]
[2024-04-02 17:41:55,131: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 17:41:55,133: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 17:41:55,134: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 17:41:55,135: INFO: common: created directory at: artifacts]
[2024-04-02 17:41:55,135: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 17:41:55,514: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 17:41:55,715: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 17:41:55,716: INFO: main: *******************]
[2024-04-02 17:41:55,716: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 17:41:55,718: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 17:41:55,720: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 17:41:55,720: INFO: common: created directory at: artifacts]
[2024-04-02 17:41:55,720: INFO: common: created directory at: artifacts\training]
[2024-04-02 17:42:51,478: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 17:42:51,478: INFO: main: *******************]
[2024-04-02 17:42:51,478: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 17:42:51,480: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 17:42:51,482: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 17:42:51,482: INFO: common: created directory at: artifacts]
[2024-04-02 17:43:00,786: INFO: common: json file saved at: scores.json]
[2024-04-02 17:43:00,786: INFO: common: json file saved at: scores.json]
[2024-04-02 17:43:00,786: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 18:11:29,521: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-02 18:11:29,522: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 18:12:48,305: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-02 18:12:48,305: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 18:13:11,212: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 18:13:11] "GET / HTTP/1.1" 200 -]
[2024-04-02 18:14:23,531: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-04-02 18:14:23,535: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 18:14:23] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-04-02 18:15:47,823: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 18:15:47] "GET / HTTP/1.1" 200 -]
[2024-04-02 20:01:31,431: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-04-02 20:01:31,433: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 20:01:31] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-04-02 20:04:30,496: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-02 20:04:30,496: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 20:04:36,637: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 20:04:36] "GET / HTTP/1.1" 200 -]
[2024-04-02 20:04:45,605: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-04-02 20:04:45,608: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 20:04:45] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-04-02 20:52:12,476: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-02 20:52:12,476: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 20:52:49,434: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 20:52:49] "GET / HTTP/1.1" 200 -]
[2024-04-02 20:53:34,233: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\prediction.py", line 16, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-04-02 20:53:34,236: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 20:53:34] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-04-02 20:56:19,055: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 20:56:19] "GET / HTTP/1.1" 200 -]
[2024-04-02 21:08:43,469: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-02 21:08:43,469: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 21:08:58,282: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\prediction.py", line 18, in predict
    model = load_model(os.path.join("model", "model.h5"))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\saving_api.py", line 212, in load_model
    return legacy_sm_saving_lib.load_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\utils\traceback_utils.py", line 70, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\keras\saving\legacy\save.py", line 230, in load_model
    raise IOError(
OSError: No file or directory found at model\model.h5
[2024-04-02 21:08:58,284: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 21:08:58] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-04-02 21:54:18,448: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 21:54:18,448: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 21:54:30,715: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 21:54:30] "GET / HTTP/1.1" 200 -]
[2024-04-02 21:54:42,696: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\prediction.py", line 21, in predict
    test_image = image.load_img(imagename, target_size = (224,224))
AttributeError: module 'keras.preprocessing.image' has no attribute 'load_img'
[2024-04-02 21:54:42,699: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 21:54:42] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-04-02 21:56:43,852: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 21:56:43,853: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 21:56:51,844: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 21:56:51] "GET / HTTP/1.1" 200 -]
[2024-04-02 21:56:59,246: ERROR: app: Exception on /predict [POST]]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\flask_cors\decorator.py", line 130, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "app.py", line 44, in predictRoute
    result = clApp.classifier.predict()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\prediction.py", line 21, in predict
    test_image = image.load_img(imagename, target_size = (224,224))
AttributeError: module 'keras.preprocessing.image' has no attribute 'load_img'
[2024-04-02 21:56:59,248: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 21:56:59] "[35m[1mPOST /predict HTTP/1.1[0m" 500 -]
[2024-04-02 21:59:58,524: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 21:59:58,524: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 22:00:06,696: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:00:06] "GET / HTTP/1.1" 200 -]
[2024-04-02 22:00:15,964: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:00:15] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:00:50,582: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:00:50] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:00:53,042: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:00:53] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:01:00,461: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:01:00] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:03:06,566: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 22:03:06,569: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:03:06,571: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:03:06,571: INFO: common: created directory at: artifacts]
[2024-04-02 22:03:06,571: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 22:03:06,571: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 22:03:20,376: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 22:03:20,815: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 22:03:20,815: INFO: main: *******************]
[2024-04-02 22:03:20,815: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 22:03:20,817: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:03:20,819: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:03:20,819: INFO: common: created directory at: artifacts]
[2024-04-02 22:03:20,819: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 22:03:21,272: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 22:03:21,494: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 22:03:21,494: INFO: main: *******************]
[2024-04-02 22:03:21,494: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 22:03:21,496: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:03:21,498: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:03:21,498: INFO: common: created directory at: artifacts]
[2024-04-02 22:03:21,498: INFO: common: created directory at: artifacts\training]
[2024-04-02 22:04:17,792: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 22:04:17,792: INFO: main: *******************]
[2024-04-02 22:04:17,792: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 22:04:17,794: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:04:17,796: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:04:17,796: INFO: common: created directory at: artifacts]
[2024-04-02 22:04:27,090: INFO: common: json file saved at: scores.json]
[2024-04-02 22:04:27,091: INFO: common: json file saved at: scores.json]
[2024-04-02 22:04:27,091: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 22:15:10,555: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 22:15:10,557: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:15:10,559: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:15:10,560: INFO: common: created directory at: artifacts]
[2024-04-02 22:15:10,560: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 22:15:10,560: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 22:15:24,592: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 22:15:25,105: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 22:15:25,105: INFO: main: *******************]
[2024-04-02 22:15:25,105: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 22:15:25,107: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:15:25,109: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:15:25,109: INFO: common: created directory at: artifacts]
[2024-04-02 22:15:25,110: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 22:15:25,521: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 22:15:25,727: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 22:15:25,728: INFO: main: *******************]
[2024-04-02 22:15:25,728: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 22:15:25,730: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:15:25,732: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:15:25,732: INFO: common: created directory at: artifacts]
[2024-04-02 22:15:25,733: INFO: common: created directory at: artifacts\training]
[2024-04-02 22:16:23,903: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 22:16:23,903: INFO: main: *******************]
[2024-04-02 22:16:23,903: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 22:16:23,905: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 22:16:23,907: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 22:16:23,908: INFO: common: created directory at: artifacts]
[2024-04-02 22:16:33,458: INFO: common: json file saved at: scores.json]
[2024-04-02 22:16:33,459: INFO: common: json file saved at: scores.json]
[2024-04-02 22:16:33,459: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 22:19:58,091: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 22:19:58,091: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 22:20:12,901: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:20:12] "GET / HTTP/1.1" 200 -]
[2024-04-02 22:20:21,294: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:20:21] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:20:33,891: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:20:33] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:20:36,546: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:20:36] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:54:47,518: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 22:54:47,518: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 22:55:05,110: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:55:05] "GET / HTTP/1.1" 200 -]
[2024-04-02 22:55:14,984: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:55:14] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:55:22,400: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:55:22] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:55:39,240: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:55:39] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:55:40,901: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:55:40] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:56:02,915: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x00000248C0FBD040> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-02 22:56:02,918: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:56:02] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:56:09,396: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x00000248C0FBD9D0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-02 22:56:09,399: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:56:09] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:56:14,708: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:56:14] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:56:35,210: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:56:35] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:56:49,490: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:56:49] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:57:04,800: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:57:04] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:58:13,914: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 22:58:13,915: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 22:58:19,590: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:58:19] "GET / HTTP/1.1" 200 -]
[2024-04-02 22:58:29,454: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:58:29] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:58:40,399: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:58:40] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 22:58:48,767: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 22:58:48] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:03:48,632: INFO: stage_03_model_training: *******************]
[2024-04-02 23:03:48,632: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-02 23:03:48,634: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:03:48,636: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:03:48,636: INFO: common: created directory at: artifacts]
[2024-04-02 23:03:48,636: INFO: common: created directory at: artifacts\training]
[2024-04-02 23:04:45,775: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 23:04:51,441: INFO: stage_04_model_evaluation: *******************]
[2024-04-02 23:04:51,441: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 23:04:51,444: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:04:51,445: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:04:51,446: INFO: common: created directory at: artifacts]
[2024-04-02 23:05:01,076: INFO: common: json file saved at: scores.json]
[2024-04-02 23:05:01,077: INFO: common: json file saved at: scores.json]
[2024-04-02 23:05:01,077: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 23:05:30,645: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 23:05:30,645: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 23:08:15,488: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:08:15] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:08:28,985: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:08:28] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:08:55,180: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:08:55] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:27:55,181: INFO: stage_03_model_training: *******************]
[2024-04-02 23:27:55,181: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-02 23:27:55,184: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:27:55,185: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:27:55,186: INFO: common: created directory at: artifacts]
[2024-04-02 23:27:55,186: INFO: common: created directory at: artifacts\training]
[2024-04-02 23:28:51,331: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 23:28:56,822: INFO: stage_04_model_evaluation: *******************]
[2024-04-02 23:28:56,822: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 23:28:56,825: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:28:56,826: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:28:56,827: INFO: common: created directory at: artifacts]
[2024-04-02 23:29:06,390: INFO: common: json file saved at: scores.json]
[2024-04-02 23:29:06,390: INFO: common: json file saved at: scores.json]
[2024-04-02 23:29:06,390: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 23:32:31,994: INFO: stage_03_model_training: *******************]
[2024-04-02 23:32:31,994: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-02 23:32:31,997: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:32:31,998: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:32:31,998: INFO: common: created directory at: artifacts]
[2024-04-02 23:32:31,999: INFO: common: created directory at: artifacts\training]
[2024-04-02 23:33:56,072: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 23:34:01,729: INFO: stage_04_model_evaluation: *******************]
[2024-04-02 23:34:01,729: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 23:34:01,731: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:34:01,733: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:34:01,733: INFO: common: created directory at: artifacts]
[2024-04-02 23:34:11,246: INFO: common: json file saved at: scores.json]
[2024-04-02 23:34:11,247: INFO: common: json file saved at: scores.json]
[2024-04-02 23:34:11,247: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 23:35:43,954: INFO: stage_03_model_training: *******************]
[2024-04-02 23:35:43,954: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-02 23:35:43,957: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:35:43,958: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:35:43,959: INFO: common: created directory at: artifacts]
[2024-04-02 23:35:43,959: INFO: common: created directory at: artifacts\training]
[2024-04-02 23:38:03,541: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 23:38:09,080: INFO: stage_04_model_evaluation: *******************]
[2024-04-02 23:38:09,080: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 23:38:09,082: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:38:09,084: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:38:09,084: INFO: common: created directory at: artifacts]
[2024-04-02 23:38:18,571: INFO: common: json file saved at: scores.json]
[2024-04-02 23:38:18,571: INFO: common: json file saved at: scores.json]
[2024-04-02 23:38:18,572: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 23:38:44,431: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.5:8084]
[2024-04-02 23:38:44,431: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-02 23:38:52,013: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:38:52] "GET / HTTP/1.1" 200 -]
[2024-04-02 23:39:18,749: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:39:18] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:39:35,047: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:39:35] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:39:38,316: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:39:38] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:39:40,412: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:39:40] "GET / HTTP/1.1" 200 -]
[2024-04-02 23:39:49,469: INFO: _internal: 127.0.0.1 - - [02/Apr/2024 23:39:49] "POST /predict HTTP/1.1" 200 -]
[2024-04-02 23:54:57,288: INFO: stage_04_model_evaluation: *******************]
[2024-04-02 23:54:57,288: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 23:54:57,290: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 23:54:57,292: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 23:54:57,292: INFO: common: created directory at: artifacts]
[2024-04-02 23:55:06,606: INFO: common: json file saved at: scores.json]
[2024-04-02 23:55:06,606: INFO: common: json file saved at: scores.json]
[2024-04-02 23:55:06,607: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-04 05:53:36,769: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-04 05:53:36,770: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-04 05:53:42,551: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:53:42] "GET / HTTP/1.1" 200 -]
[2024-04-04 05:53:55,847: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:53:55] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:54:04,483: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:54:04] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:56:35,954: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:56:35] "GET / HTTP/1.1" 200 -]
[2024-04-04 05:56:54,233: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:56:54] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:57:11,449: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:57:11] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:57:14,964: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:57:14] "GET / HTTP/1.1" 200 -]
[2024-04-04 05:57:23,619: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x0000018E59CEDCA0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-04 05:57:23,626: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:57:23] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:57:43,867: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x0000018E59CED430> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-04 05:57:43,875: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:57:43] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:57:54,925: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:57:54] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:58:11,173: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:58:11] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:58:21,012: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:58:21] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:58:40,646: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:58:40] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 05:58:49,733: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 05:58:49] "POST /predict HTTP/1.1" 200 -]
[2024-04-04 10:35:18,654: INFO: _internal: 127.0.0.1 - - [04/Apr/2024 10:35:18] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:15:11,240: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.4:8084]
[2024-04-09 14:15:11,241: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-09 14:15:14,546: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:15:14] "GET / HTTP/1.1" 200 -]
[2024-04-09 14:15:46,757: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:15:46] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:16:01,401: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:16:01] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:16:13,335: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:16:13] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:17:24,272: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:17:24] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:17:34,595: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x00000294C151E040> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-09 14:17:34,598: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:17:34] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:18:29,007: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x00000294C151E9D0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-09 14:18:29,010: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:18:29] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:19:17,604: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:19:17] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:19:26,600: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:19:26] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:30:04,458: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:30:04] "POST /predict HTTP/1.1" 200 -]
[2024-04-09 14:57:01,147: INFO: _internal: 127.0.0.1 - - [09/Apr/2024 14:57:01] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 17:34:38,351: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-15 17:34:38,354: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 17:34:38,357: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 17:34:38,357: INFO: common: created directory at: artifacts]
[2024-04-15 17:34:38,358: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-15 17:34:38,358: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-15 17:34:52,175: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-15 17:34:52,566: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-15 17:34:52,566: INFO: main: *******************]
[2024-04-15 17:34:52,566: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-15 17:34:52,569: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 17:34:52,571: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 17:34:52,571: INFO: common: created directory at: artifacts]
[2024-04-15 17:34:52,571: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-15 17:34:53,059: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-15 17:34:53,281: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-15 17:34:53,281: INFO: main: *******************]
[2024-04-15 17:34:53,281: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-15 17:34:53,284: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 17:34:53,286: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 17:34:53,286: INFO: common: created directory at: artifacts]
[2024-04-15 17:34:53,286: INFO: common: created directory at: artifacts\training]
[2024-04-15 17:37:14,820: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-15 17:37:14,820: INFO: main: *******************]
[2024-04-15 17:37:14,820: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-15 17:37:14,822: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 17:37:14,824: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 17:37:14,824: INFO: common: created directory at: artifacts]
[2024-04-15 17:37:24,330: INFO: common: json file saved at: scores.json]
[2024-04-15 17:37:24,330: INFO: common: json file saved at: scores.json]
[2024-04-15 17:37:24,331: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-15 17:39:16,782: INFO: stage_03_model_training: *******************]
[2024-04-15 17:39:16,782: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-15 17:39:16,785: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 17:39:16,786: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 17:39:16,787: INFO: common: created directory at: artifacts]
[2024-04-15 17:39:16,787: INFO: common: created directory at: artifacts\training]
[2024-04-15 17:44:27,376: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-15 17:44:32,996: INFO: stage_04_model_evaluation: *******************]
[2024-04-15 17:44:32,996: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-15 17:44:32,999: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 17:44:33,000: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 17:44:33,001: INFO: common: created directory at: artifacts]
[2024-04-15 17:44:43,071: INFO: common: json file saved at: scores.json]
[2024-04-15 17:44:43,072: INFO: common: json file saved at: scores.json]
[2024-04-15 17:44:43,072: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-15 17:45:24,443: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.4:8084]
[2024-04-15 17:45:24,443: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-15 17:46:04,453: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 17:46:04] "GET / HTTP/1.1" 200 -]
[2024-04-15 17:46:08,497: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 17:46:08] "GET / HTTP/1.1" 200 -]
[2024-04-15 17:46:19,779: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 17:46:19] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 17:52:12,634: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 17:52:12] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 17:52:23,424: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 17:52:23] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 17:52:31,023: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 17:52:31] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 18:30:05,102: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x000001F441A5C040> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-15 18:30:05,105: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 18:30:05] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 18:30:58,042: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x000001F441A5C9D0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-15 18:30:58,045: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 18:30:58] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 18:50:39,792: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-15 18:50:39,794: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 18:50:39,796: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 18:50:39,796: INFO: common: created directory at: artifacts]
[2024-04-15 18:50:39,797: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-15 18:50:39,797: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-15 18:50:55,960: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-15 18:50:56,360: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-15 18:50:56,360: INFO: main: *******************]
[2024-04-15 18:50:56,360: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-15 18:50:56,362: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 18:50:56,364: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 18:50:56,364: INFO: common: created directory at: artifacts]
[2024-04-15 18:50:56,364: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-15 18:50:56,758: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-15 18:50:56,948: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-15 18:50:56,948: INFO: main: *******************]
[2024-04-15 18:50:56,948: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-15 18:50:56,950: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 18:50:56,952: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 18:50:56,953: INFO: common: created directory at: artifacts]
[2024-04-15 18:50:56,953: INFO: common: created directory at: artifacts\training]
[2024-04-15 19:00:18,605: INFO: stage_03_model_training: *******************]
[2024-04-15 19:00:18,606: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-15 19:00:18,608: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:00:18,610: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:00:18,611: INFO: common: created directory at: artifacts]
[2024-04-15 19:00:18,611: INFO: common: created directory at: artifacts\training]
[2024-04-15 19:03:02,761: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-15 19:03:02,764: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:03:02,765: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:03:02,766: INFO: common: created directory at: artifacts]
[2024-04-15 19:03:02,766: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-15 19:03:02,766: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-15 19:03:20,954: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-15 19:03:21,330: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-15 19:03:21,330: INFO: main: *******************]
[2024-04-15 19:03:21,330: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-15 19:03:21,332: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:03:21,334: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:03:21,334: INFO: common: created directory at: artifacts]
[2024-04-15 19:03:21,334: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-15 19:03:21,706: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-15 19:03:21,906: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-15 19:03:21,906: INFO: main: *******************]
[2024-04-15 19:03:21,906: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-15 19:03:21,909: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:03:21,910: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:03:21,911: INFO: common: created directory at: artifacts]
[2024-04-15 19:03:21,911: INFO: common: created directory at: artifacts\training]
[2024-04-15 19:05:12,668: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-04-15 19:05:12,671: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:05:12,673: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:05:12,673: INFO: common: created directory at: artifacts]
[2024-04-15 19:05:12,673: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-15 19:05:12,673: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-15 19:06:23,882: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-04-15 19:06:23,884: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:06:23,886: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:06:23,886: INFO: common: created directory at: artifacts]
[2024-04-15 19:06:23,886: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-15 19:06:23,886: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-15 19:08:15,177: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-15 19:08:19,111: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x=========x]
[2024-04-15 19:08:43,821: INFO: stage_02_prepare_base_model: *******************]
[2024-04-15 19:08:43,821: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-15 19:08:43,824: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:08:43,825: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:08:43,826: INFO: common: created directory at: artifacts]
[2024-04-15 19:08:43,826: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-15 19:08:44,223: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-15 19:08:44,414: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-15 19:08:51,101: INFO: stage_03_model_training: *******************]
[2024-04-15 19:08:51,101: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-15 19:08:51,104: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 19:08:51,105: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 19:08:51,106: INFO: common: created directory at: artifacts]
[2024-04-15 19:08:51,106: INFO: common: created directory at: artifacts\training]
[2024-04-15 21:14:44,037: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-15 21:14:53,608: INFO: stage_04_model_evaluation: *******************]
[2024-04-15 21:14:53,608: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-15 21:14:53,610: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-15 21:14:53,612: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-15 21:14:53,612: INFO: common: created directory at: artifacts]
[2024-04-15 21:16:22,822: INFO: common: json file saved at: scores.json]
[2024-04-15 21:16:22,823: INFO: common: json file saved at: scores.json]
[2024-04-15 21:16:22,823: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-15 21:17:00,490: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.1.4:8084]
[2024-04-15 21:17:00,490: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-15 21:17:14,726: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:17:14] "GET / HTTP/1.1" 200 -]
[2024-04-15 21:17:31,816: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:17:31] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:17:43,078: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:17:43] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:18:18,530: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:18:18] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:18:30,406: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:18:30] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:18:46,726: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x00000283663CE040> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-15 21:18:46,729: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:18:46] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:19:51,897: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x00000283663CE9D0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2024-04-15 21:19:51,900: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:19:51] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:20:05,422: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:20:05] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:20:24,776: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:20:24] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:20:36,958: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:20:36] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:20:48,563: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:20:48] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:21:06,553: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:21:06] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:21:23,134: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:21:23] "POST /predict HTTP/1.1" 200 -]
[2024-04-15 21:21:45,110: INFO: _internal: 127.0.0.1 - - [15/Apr/2024 21:21:45] "POST /predict HTTP/1.1" 200 -]
[2024-04-18 15:25:19,637: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-18 15:25:19,637: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-18 15:25:19,672: INFO: _internal: 127.0.0.1 - - [18/Apr/2024 15:25:19] "GET / HTTP/1.1" 200 -]
[2024-04-18 15:32:09,306: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8084
 * Running on http://192.168.29.47:8084]
[2024-04-18 15:32:09,307: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-04-18 15:32:10,575: INFO: _internal: 127.0.0.1 - - [18/Apr/2024 15:32:10] "GET / HTTP/1.1" 200 -]
[2024-04-18 15:32:33,594: INFO: _internal: 127.0.0.1 - - [18/Apr/2024 15:32:33] "POST /predict HTTP/1.1" 200 -]
[2024-04-18 20:59:03,392: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.29.47:8080]
[2024-04-18 20:59:03,392: INFO: _internal: [33mPress CTRL+C to quit[0m]
